spec/support/jasmine.json

```diff
@@ -0,0 +1,11 @@
+{
+  "spec_dir": "src",
```

Comment:
Wait! Why not in spec directory? This is noise.

Reply:
I can see that this pattern of test can be very noisy, but how many files are we going to have here. And anyway, it would be easy to move *.test.js files out to separate dir later.

Since, the file count is relatively small, having tests juxtaposed to code files helps in better development. 2 cents.

Reply:
I am going with @gitfaf here. It's just for better visibility and prodding developer to test code. And tests will be hard to miss; if missed, you can figure out quickly.